arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
     "description": "Composite Swagger for Compute Client"
   },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/containerService.json"
+    "./arm-compute/2016-03-30/swagger/compute.json",
+    "./arm-compute/2016-03-30/swagger/containerService.json"
   ]
```

Comment:
On this only @devigned can make the final decision. I adapted my script in a feature branch ready to be merged in case of.